packages/quantic/docs/adding-tests.md

```diff
@@ -0,0 +1,138 @@
+
+<!-- Which outputs: Hello Alex! -->
+```
+
+## Updating the Quantic Examples community
+
+Now you wish to make this component available in the `Quantic Examples` community and add end to end tests to prevent any regression. To achieve that, you want a community page containing your example component, and a configuration panel allowing to modify the `name` option.
+
+Luckily, there are some components that can help you.
+
+- the `exampleLayout` LWC component acts as a master page, so all component pages can share the same structure and style.
+- the `configurator` LWC component renders a form allowing the user to specify option values. You can create your own configurator component, but the generic one works for most cases.
+
+Now you need to create an LWC component that ties everything together. Here is the resulting `exampleQuanticGreeting` component:
+
+```html
+<!-- force-app/examples/main/lwc/exampleQuanticGreeting/exampleQuanticGreeting.html -->
+
+<template>
+    <c-example-layout
+        title={pageTitle}
+        description={pageDescription}
+        show-preview={isConfigured}>
+
+        <c-configurator slot="configuration" options={options} ontryitnow={handleTryItNow}></c-configurator>
+
+        <c-quantic-greeting slot="preview" name={config.name}></c-quantic-greeting>
+    </c-example-layout>
+</template>
+```
+
+```javascript
+// force-app/examples/main/lwc/exampleQuanticGreeting/exampleQuanticGreeting.js
+
+import {LightningElement, track} from 'lwc';
+
+export default class ExampleQuanticGreeting extends LightningElement {
+    // `config` stores the options retrieved from the configurator. 
+    @track config = {}
+    isConfigured = false;
+
+    pageTitle = 'Quantic Greeting';
+    pageDescription = 'The Quantic Greeting renders a greeting message given a name.';
+
+    // `options` is used by `configurator` to render the configuration form.
+    options = [
+        {
+            attribute: 'name',
+            label: 'Name',
+            description: 'The name to display in the greeting message.',
+            defaultValue: 'World'
+        }
+    ];
+
+    handleTryItNow(evt) {
+        this.config = evt.detail;
+
+        // Setting this to `true` makes the `preview` slot visible,
+        // which also loads the Quantic component with the configured options.
+        this.isConfigured = true;
+    }
+}
+```
+
+```xml
+<!-- force-app/examples/main/lwc/exampleQuanticGreeting/exampleQuanticGreeting.js-meta.xml -->
+
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>50.0</apiVersion>
+    <isExposed>true</isExposed>
+    <targets>
+      <target>lightning__RecordPage</target>
+      <target>lightning__AppPage</target>
+      <target>lightning__HomePage</target>
+      <target>lightningCommunity__Page</target>
+      <target>lightningCommunity__Default</target>
+  </targets>
+</LightningComponentBundle>
+```
+
```

Comment:
I think the following steps would be clearer if they came out as a list. I would add an introduction sentence, and then use a numbered list. 

```
Next, execute the following steps:

1. Deploy the example components to your org.
2. Open the community builder, then create a `Quantic Greeting` page and drop your `exampleQuanticGreeting` component into it.
...
```